### ISSUE TITLE
Unable to disable disk metrics in version 0.25.0 (same as issue #1267)

### ISSUE THREAD

**Issue by msinakitajr** [google/cadvisor#1661]
Using --disable_metrics=disk, I was expecting that the file system metrics (container_fs_xxx) would not appear but they still did.  I'm trying to exclude a set of disk volumes from cadvisor's "namespace.

**Comment by dashpole** [comment]
Those could be diskio metrics, and not disk metrics.  Are you seeing container_fs_inodes_xxx or container_fs_usage_xxxx?  AFAIK, we dont have the ability to disable diskio metrics.

**Comment by msinakitajr** [comment]
Yes, I can see the container_fs_inodes_xxx as well as the container_fs_usage_xxx metrics.
Is there anyway to exclude the disk device namespace?

**Comment by dashpole** [comment]
Ill look into this.

**Comment by msinakitajr** [comment]
thanks dashpole!

**Comment by ronenn1** [comment]
+1

**Comment by nunofernandes** [comment]
Any news on this? My servers' load increases dramatically after starting cadvisor (due to disk metrics).

Thanks

**Comment by dashpole** [comment]
do those disk metrics have non-zero values?  Something I fixed recently is that we still omit metrics (with values of 0) for disabled metrics.

**Comment by dims** [comment]
Closing this issue as it has been open since before January 1, 2024. If this is still relevant, please feel free to reopen or create a new issue.
